### ISSUE TITLE
Weppy CLI gives error on Windows

### ISSUE THREAD

**Issue by BrenBarn** [gi0baro/weppy#213]
Trying to use `weppy --app=myapp run` on Windows gives an error like this:

```
> Serving weppy application myapp
> weppy application myapp running on http://127.0.0.1:8000 (press CTRL+C to quit)
> Restarting (stat mode)
  File "C:\FakeProgs\Anaconda3\envs\weppy\Scripts\weppy.exe", line 1
SyntaxError: Non-UTF-8 code starting with '\x90' in file C:\FakeProgs\Anaconda3\envs\weppy\Scripts\weppy.exe on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

It appears to be the same issue described [here](https://github.com/pallets/flask/issues/2543) for Flask, which apparently was a duplicate of a werkzeug issue [here](https://github.com/pallets/werkzeug/issues/1136).  It seems that on Windows the script is an .exe, and trying to re-run that causes Python to think it's a Python script with invalid syntax.  Given that a comment in the weppy CLI source file says it was based on the code from Flask, perhaps a similar fix can be applied here.

In the meantime, I was able to work around it by using `python -m weppy.cli --app=myapp run`.  Would be nice to have it fixed though.

**Comment by gi0baro** [comment]
Hi @BrenBarn,
thank you for reporting this and the useful references.
I will push an update in the next few days.

**Comment by gi0baro** [comment]
This is fixed in 1.2.9
